pkg/cloud/aws/aws.go

```diff
@@ -146,6 +146,21 @@ func (c *AwsCloud) CreateEC2Instances(count int, amiID, instanceType, keyName, s
 				},
 			},
 		},
+		TagSpecifications: []types.TagSpecification{
```

Comment:
can we do the same with static ip names?

Reply:
sure